src/platform/mhv/secondary-nav/containers/MhvSecondaryNav.jsx

```diff
@@ -25,7 +25,7 @@ export const mhvSecNavItems = [
     title: 'Messages',
     actionName: `${actionPrefix} - Messages`,
     icon: 'forum',
-    href: `/my-health/secure-messages`,
+    href: `/my-health/secure-messages/inbox`,
```

Comment:
Just need to add below line because the `appRootUrl` is required if the value differs from `href`s:
`appRootUrl: '/my-health/secure-messages',`

Reply:
done, lmk if it needs to be changed